spec/Powertrain/Battery.vspec

```diff
@@ -8,9 +8,22 @@
 #
 
 #
-# Signals, attributes and actuators around the electric powertrain.
+# Signals and attributes related to the traction battery in vehicles with electrical powertrain.
+# For some vehicles the traction battery is referred to as the high voltage battery.
 #
 
+# It might be possible to disconnect the traction battery used by an electric powertrain.
+# This is achieved by connectors, typically one for plus and one for minus.
+PowerConnected:
```

Comment:
We got some additional information internally on that connectors for power/plus and ground/minus are handled separately in at least some vehicles, to handle different use cases. A typical example is:

- Vehicle off, no charging: Both connectors disconnected/open
- Vehicle off, charging: Ground connector connected/closed, Power connector disconnected/open
- Vehicle on: Both connectors connected/closed

Based on this we propose to introducing two separate boolean sensor signals.